pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.2</version>
+            <version>2.4-SNAPSHOT</version> <!-- Snapshot until enforced ordering in DepthFirstScanner is released -->
```

Comment:
FWIW I find it helpful to link to the actual upstream PR—simplifies bookkeeping when there are numerous PRs in flight.